src/components/SearchResults/SearchResutls.test.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { IntlProvider } from 'react-intl';
+import { MemoryRouter } from 'react-router-dom';
+import SearchResults from '.';
+import {
+  FILTER_ID_ALL, FILTER_ID_DOCUMENTS, FILTER_ID_IMAGES, FILTER_ID_VIDEOS,
+} from '../../data/constants/filterTypes';
+
+
+import { mockGetResultCount, mockSearchResults } from './SearchResults.mock';
+
+const SearchResultWrapper = props => (
+  <IntlProvider locale="en">
+    <MemoryRouter>
+      <SearchResults
+        {...props}
+      />
+    </MemoryRouter>
+  </IntlProvider>
+);
+
+describe('<SearchResults/>', () => {
+  describe('renders', () => {
+    it('with no searchResults', () => {
+      const tree = renderer
+        .create((
+          <SearchResultWrapper filter={FILTER_ID_ALL} />
+        ))
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('with filter all', () => {
+      const tree = renderer
+        .create((
+          <SearchResultWrapper
+            resultsCounter={mockGetResultCount}
```

Comment:
nit: can we pass `resultsCounter` and `hits` at the time we define `SearchResultWrapper ` to avoid duplication in tests.

Reply:
but in some tests we do not want to pass these params, for example test `renders with no searchResults`